src/hotspot/share/oops/instanceKlass.cpp

```diff
@@ -3824,9 +3830,9 @@ void InstanceKlass::print_class_load_logging(ClassLoaderData* loader_data,
     // Classfile checksum
     if (cfs) {
       debug_stream.print(" bytes: %d checksum: %08x",
-                         cfs->length(),
-                         ClassLoader::crc32(0, (const char*)cfs->buffer(),
-                         cfs->length()));
+                        cfs->length(),
+                        ClassLoader::crc32(0, (const char*)cfs->buffer(),
+                        cfs->length()));
```

Comment:
The indentation changes you made here (and elsewhere) are incorrect - the original was correct: arguments should line up

Reply:
Sorry about that. I'll piggy back a fix onto https://github.com/openjdk/jdk/pull/14851.